applications/asset_tracker_v2/src/events/app_module_event.c

```diff
@@ -110,7 +110,10 @@ COMMON_EVENT_INFO_DEFINE(app_module_event,
 
 #endif /* CONFIG_PROFILER */
 
+static const uint8_t app_module_event_flags = (IS_ENABLED(CONFIG_APP_EVENTS_LOG))
```

Comment:
```
static const uint8_t app_module_event_flags = (IS_ENABLED(CONFIG_APP_EVENTS_LOG)) ?
                                                                           BIT(EVENT_FLAGS_INIT_LOG_ENABLE) : 0;
```
Consider moving `?` to previous line and aligning (for consistency with existing NCS code)